atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleOverflowWriteTable.java

```diff
@@ -216,6 +219,60 @@ public void delete(List<Entry<Cell, Long>> entries) {
                 args);
     }
 
+    @Override
+    public void delete(RangeRequest range) {
+        String shortTableName = getShortTableName();
+
+        switch (config.overflowMigrationState()) {
```

Comment:
super nit: but this logic is dupped from above and could be abstracted away and just have the delete function passed in.